frontend/public/manifest.json

```diff
@@ -1,6 +1,6 @@
 {
-  "short_name": "React App",
-  "name": "Create React App Sample",
+  "short_name": "CCBC",
+  "name": "CCBC",
```

Comment:
should the long name maybe be `Canadian Children's Book Center`?

Reply:
Good point